ui/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.4'
+services:
+  ui:
+    build:
+      context: ..
+      dockerfile: ui/Dockerfile
+    ports:
+    - "${PORT}:${PORT}"
+    command: 'true'
```

Comment:
my expectation is that `command` is not needed if `entrypoint` specified

Reply:
Good spot. Looks like the same thing happens in our `oracle/docker-compose.yml`. Are you OK with creating a separate issue and PR for this in order to not mix up work and keep PRs small?

Reply:
We can create the issue to remove this line from existing `oracle/docker-compose.yml`. And seems that we introduce the new file in this PR, I would get rid of double work by committing correct version of the file. Do you see any objections?

Reply:
Totally agree, I forgot that the file is being introduced and not already existing.
I created #106 for `oracle/docker-compose.yml`

Reply:
thanks!